src/components/SeButton.jsx

```diff
@@ -14,7 +14,7 @@ const SeButton = (props) => {
     ...rest
   } = props;
   const cssClasses = classNames(
-    `${isDisabled ? "disabled" : ""}`,
+    `d-sm-block mb-3 ${isDisabled ? "disabled" : ""}`,
```

Comment:
a couple comments about classnames, here is their [docs](https://github.com/baltimorecounty/react-baltcogo/pull/214/files) for more info

this can be cleaned up a bit


```js
const cssClasses = classNames(
  "d-sm-block mb-3",
  className,
  { "disabled" : isDisabled },
  { "is-loading": isLoading }
);
```
This makes it a little easier to read, and removes the duplicate disabled class.

@theilman7 just an fyi

Reply:
yeah I did this to remove all those class includes for seButton

Reply:
@tmccoy529 i realized the duplicate was a copied line, so ignore that piece of the comment.

Reply:
this is done